lang/rust/Portfile

```diff
@@ -24,8 +24,8 @@ long_description    Rust is a curly-brace, block-structured expression \
 homepage            https://www.rust-lang.org/
 
 # can use cmake or cmake-devel; default to cmake.
-depends_build       bin:python2.7:python27 \
-                    bin:cmake:cmake
+depends_build       path:bin/cmake:cmake \
+                    bin:python2.7:python27
```

Comment:
Is rust patched to use our `python2.7`?

Reply:
Unrelated to this PR.

Reply:
Yes, but the question still remains.

Reply:
If we want to be pedantic, I would have required another PR to change the ordering of cmake with python but I didn't think we did that kind of review here.

Reply:
Well, it made the order consistent with the cargo port, and with our convention of alphabetizing dependencies.